### ISSUE TITLE
Very easy request, edit view stays in edit view, even when navigating to other servers.

### ISSUE THREAD

**Issue by jaxjexjox** [louislam/uptime-kuma#3419]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Hello and thanks for the hard work!

I often click edit, to check how I have configured something (example the 'retries' before issuing an alert)
I adjust the retries and click save which is handy.
I would then lick to click on another entry, on the left hand column and *continue to see the edit view, for the new entry*

Instead it goes to the front entry and I then need to click edit, again.

Here's a very rough pic:
https://i.imgur.com/EoSMtM6.png

### ✔️ Solution

Basically if you're in the EDIT ENTRY view and you click on a different entry on the left column to work with, I'd like to not have to click "edit" again.

### ❓ Alternatives

There's no real alternative truth be told, this is the quickest and easiest solution, unless you break the entire data out in to some kind of highly detailed looking table (almost like excel?) so I can see all entries at a glance.  That would also be lovely.

I will paypal $20 USD for this request.
Regardless thank you for the hard work, really, it's wonderful and I appreciate your efforts.

**Comment by CommanderStorm** [comment]
I am not convinced that that would be transparent to the user.
Users commonly tab outside of a modal to get out of this window
=> I would argue that the edit window should behave the same as if this were a modal, given how it animates up.

What you most likely want is a [bulk-edit option](https://github.com/louislam/uptime-kuma/issues?q=is%3Aissue+is%3Aopen+bulk+edit).

**Comment by jaxjexjox** [comment]
You could add a simple "close" next to the save button down the bottom, which would THEN finally close the edit view, regardless of which server entry you'd clicked and worked on.

The bulk edit option, would probably also be viable, assuming the interface was big enough to work with.

I would like the easiest solution, for the developers at the end of the day.

**Comment by jaxjexjox** [comment]
> I am not convinced that that would be transparent to the user. Users commonly tab outside of a modal to get out of this window => I would argue that the edit window should behave the same as if this were a modal, given how it animates up.
> 
> What you most likely want is a [bulk-edit option](https://github.com/louislam/uptime-kuma/issues?q=is%3Aissue+is%3Aopen+bulk+edit).

I've changed my mind.

You're right. I have another use this morning for a bulk edit option, which would also eliminate my first issue as well.

I will  bump the thread for bulk edit.

Thanks for your reply!

**Comment by CommanderStorm** [comment]
> which would also eliminate my first issue as well

I think this is a duplicate of https://github.com/louislam/uptime-kuma/issues?q=is%3Aissue+is%3Aopen+bulk+edit
If you agree, could you please close this Issue, as duplicates only create immortal zombies and are really hard to issue-manage?
If not, what makes this issue unique enough to require an additional issue? (Could this be integrated into the issue linked above?) ^^

**Comment by jaxjexjox** [comment]
Yes my intention here is that the bulk edit does infact sound like a better option overall.
